src/arch/sparcv9/CodeGen.zig

```diff
@@ -418,7 +420,7 @@ fn gen(self: *Self) !void {
         .data = .{ .dbg_line_column = .{
             .line = self.end_di_line,
             .column = self.end_di_column,
-        } },
+         } },
```

Comment:
Missing `zig fmt`?

Reply:
Hmm, that is what `zig fmt` suggested to me, should I change it back?

Reply:
What I meant is that commits that come after this should not fix things; instead, all commits should only make the changes that are necessary.